lib/src/utils/exclude_utils.dart

```diff
@@ -11,3 +11,12 @@ Iterable<Glob> prepareExcludes(Iterable<String> patterns, String root) =>
     patterns
         .map((exclude) => Glob(p.join(root, exclude).replaceAll(r'\', '/')))
         .toList();
+
+Iterable<Glob> prepareNormalizedExcludes(
```

Comment:
@incendial after introduce prepareNormalizedExcludes, do we really need prepareExcludes ?

Reply:
Replaced `prepareExcludes` with `prepareNormalizedExcludes` and kept old name.